packages/table/src/cell/formats/truncatedFormat.tsx

```diff
@@ -12,6 +12,10 @@ import * as React from "react";
 
 import * as Classes from "../../common/classes";
 
+// amount in pixels that the content div width changes when truncated vs when
+// not truncated. Note: could be modified by styles
+const CONTENT_DIV_WIDTH_DELTA = 25;
```

Comment:
i'm not so sure about that... @gscshoyru explains above that this value is basically the size of the ellipsis at our current font size. i'm asking for a prop along the lines of `Popover` `arrowSize` so you can adjust the JS value to match your actual CSS.